iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -47,6 +47,10 @@ public String getId() {
         return id;
     }
 
+    void replaceId(String newId) {
```

Comment:
To move to the API (could be later) because it is a very useful feature on its own.